### ISSUE TITLE
sscanf error: System not initialised.

### ISSUE THREAD

**Issue by thecodeah** [Southclaws/sampctl#183]
using the sscanf function gives me the error message "sscanf error: System not initialised.". This issue does not occur on a server without sampctl, thus I believe this might be a bug in sampctl.

I am running Ubuntu 18.04 LTS

server pawn.json file :
```{
	"user": "Southclaws",
	"repo": "samp-sandbox",
	"entry": "gamemodes/sandbox.pwn",
	"output": "gamemodes/sandbox.amx",
	"dependencies": [
		"sampctl/samp-stdlib",
		"Southclaws/samp-logger",
		"Southclaws/pawn-errors",
		"pawn-lang/YSI-Includes",
		"ScavengeSurvive/settings",
		"Southclaws/pawn-requests:0.8.2",
		"lassir/bcrypt-samp",
		"Awsomedude/easyDialog",
		"BigETI/pawn-map",
		"JaTochNietDan/SA-MP-FileManager",
		"maddinat0r/sscanf"
	],
	"local": true,
	"builds": [
		{
			"name": "dev",
			"includes": [
				"gamemodes/sandbox/player-lifecycle",
				"gamemodes/sandbox/player-auth",
				"gamemodes/sandbox/class-selection",
				"gamemodes/sandbox/vehicle"
			],
			"constants": {
				"NO_AUTH": "1"
			}
		},
		{
			"name": "prod",
			"includes": [
				"gamemodes/sandbox/player-lifecycle",
				"gamemodes/sandbox/player-auth",
				"gamemodes/sandbox/class-selection"
			]
		}
	],
	"runtime": {
		"version": "0.3.7",
		"mode": "server",
		"echo": "loading server.cfg generated by sampctl - do not edit this file manually, edit samp.json instead!",
		"rcon_password": "password",
		"port": 7777
	},
	"runtimes": [
		{
			"name": "main",
			"version": "0.3.7",
			"mode": "server",
			"port": 7777
		},
		{
			"name": "testing",
			"version": "0.3.7",
			"mode": "y_testing",
			"port": 7777
		}
	]
}
```

sampctl p run --verbose output :
```INFO: Southclaws/samp-sandbox read package from directory /home/codeah/Desktop/samp-sandbox
INFO: Southclaws/samp-sandbox resolving dependencies during package load
INFO: Southclaws/samp-sandbox resolving dependency tree into a flattened list...
INFO: sampctl/samp-stdlib read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/samp-stdlib
INFO: sampctl/pawn-stdlib read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/pawn-stdlib
INFO: Southclaws/samp-logger read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/samp-logger
INFO: Zeex/samp-plugin-crashdetect:v4.19 read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/samp-plugin-crashdetect
INFO: Zeex/samp-plugin-crashdetect:v4.19 adding resource include path /home/codeah/Desktop/samp-sandbox/dependencies/extracted-asset-resources/samp-plugin-crashdetect-fa6aeabe3a23d6b9335ac02fbf6b170e
INFO: Zeex/samp-plugin-crashdetect:v4.19 ignoring platform mismatch windows
INFO: Southclaws/samp-sandbox adding plugin from package runtime Zeex/samp-plugin-crashdetect as github.com/Zeex/samp-plugin-crashdetect:v4.19
INFO: Southclaws/pawn-errors:1.2.1 read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/pawn-errors
INFO: pawn-lang/YSI-Includes read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/YSI-Includes
INFO: Zeex/amx_assembly read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/amx_assembly
INFO: ScavengeSurvive/settings read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/settings
INFO: Southclaws/samp-ini:1.1.0 read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/samp-ini
INFO: BigETI/pawn-map read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/pawn-map
INFO: BigETI/pawn-memory read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/pawn-memory
INFO: BigETI/pawn-memory ignoring platform mismatch windows
INFO: Southclaws/samp-sandbox adding plugin from package runtime BigETI/pawn-memory as github.com/BigETI/pawn-memory
INFO: Southclaws/pawn-requests:0.8.2 read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/pawn-requests
INFO: Southclaws/pawn-requests:0.8.2 adding resource include path /home/codeah/Desktop/samp-sandbox/dependencies/extracted-asset-resources/pawn-requests-50e050dd1fd0383faf82f6ca80355fa7
INFO: Southclaws/pawn-requests:0.8.2 ignoring platform mismatch windows
INFO: Southclaws/samp-sandbox adding plugin from package runtime Southclaws/pawn-requests as github.com/Southclaws/pawn-requests:0.8.2
INFO: lassir/bcrypt-samp read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/bcrypt-samp
INFO: lassir/bcrypt-samp ignoring platform mismatch windows
INFO: Southclaws/samp-sandbox adding plugin from package runtime lassir/bcrypt-samp as github.com/lassir/bcrypt-samp
INFO: Awsomedude/easyDialog read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/easyDialog
INFO: BigETI/pawn-map read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/pawn-map
INFO: JaTochNietDan/SA-MP-FileManager read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/SA-MP-FileManager
INFO: JaTochNietDan/SA-MP-FileManager ignoring platform mismatch windows
INFO: Southclaws/samp-sandbox adding plugin from package runtime JaTochNietDan/SA-MP-FileManager as github.com/JaTochNietDan/SA-MP-FileManager
INFO: maddinat0r/sscanf read package from directory /home/codeah/Desktop/samp-sandbox/dependencies/sscanf
INFO: maddinat0r/sscanf adding resource include path /home/codeah/Desktop/samp-sandbox/dependencies/extracted-asset-resources/sscanf-7b57265c5a038f865e81e8e83f95f5cf
INFO: maddinat0r/sscanf ignoring platform mismatch windows
INFO: Southclaws/samp-sandbox adding plugin from package runtime maddinat0r/sscanf as github.com/maddinat0r/sscanf
INFO: read plugin from dependency: github.com/Zeex/samp-plugin-crashdetect:v4.19
INFO: read plugin from dependency: github.com/BigETI/pawn-memory
INFO: read plugin from dependency: github.com/Southclaws/pawn-requests:0.8.2
INFO: read plugin from dependency: github.com/lassir/bcrypt-samp
INFO: read plugin from dependency: github.com/JaTochNietDan/SA-MP-FileManager
INFO: read plugin from dependency: github.com/maddinat0r/sscanf
INFO: [github.com/Zeex/samp-plugin-crashdetect:v4.19 github.com/BigETI/pawn-memory github.com/Southclaws/pawn-requests:0.8.2 github.com/lassir/bcrypt-samp github.com/JaTochNietDan/SA-MP-FileManager github.com/maddinat0r/sscanf]
INFO: plugin github.com/Zeex/samp-plugin-crashdetect:v4.19 is a package dependency
INFO: getting plugin resource from cache github.com/Zeex/samp-plugin-crashdetect:v4.19 /home/codeah/.samp/plugins/samp-plugin-crashdetect/v4.19
INFO: github.com/Zeex/samp-plugin-crashdetect:v4.19 retrieved package to file: /home/codeah/.samp/plugins/samp-plugin-crashdetect/v4.19/crashdetect-4.19-linux.tar.gz
INFO: plugin github.com/BigETI/pawn-memory is a package dependency
INFO: getting plugin resource from cache github.com/BigETI/pawn-memory /home/codeah/.samp/plugins/pawn-memory/latest
INFO: github.com/BigETI/pawn-memory retrieved package to file: /home/codeah/.samp/plugins/pawn-memory/latest/pawn-memory-linux-x86.zip
INFO: plugin github.com/Southclaws/pawn-requests:0.8.2 is a package dependency
INFO: getting plugin resource from cache github.com/Southclaws/pawn-requests:0.8.2 /home/codeah/.samp/plugins/pawn-requests/0.8.2
INFO: github.com/Southclaws/pawn-requests:0.8.2 retrieved package to file: /home/codeah/.samp/plugins/pawn-requests/0.8.2/pawn-requests-linux.zip
INFO: plugin github.com/lassir/bcrypt-samp is a package dependency
INFO: getting plugin resource from cache github.com/lassir/bcrypt-samp /home/codeah/.samp/plugins/bcrypt-samp/latest
INFO: github.com/lassir/bcrypt-samp retrieved package to file: /home/codeah/.samp/plugins/bcrypt-samp/latest/bcrypt-samp-v2.2.3-debian_7_x86.tar.gz
INFO: plugin github.com/JaTochNietDan/SA-MP-FileManager is a package dependency
INFO: getting plugin resource from cache github.com/JaTochNietDan/SA-MP-FileManager /home/codeah/.samp/plugins/SA-MP-FileManager/latest
INFO: github.com/JaTochNietDan/SA-MP-FileManager retrieved package to file: /home/codeah/.samp/plugins/SA-MP-FileManager/latest/FileManager_1.5_Linux.zip
INFO: plugin github.com/maddinat0r/sscanf is a package dependency
INFO: getting plugin resource from cache github.com/maddinat0r/sscanf /home/codeah/.samp/plugins/sscanf/latest
INFO: github.com/maddinat0r/sscanf retrieved package to file: /home/codeah/.samp/plugins/sscanf/latest/sscanf-2.8.2-linux.tar.gz
INFO: adding runtime plugin crashdetect
INFO: adding runtime plugin pawn-memory
INFO: adding runtime plugin requests
INFO: adding runtime plugin bcrypt-samp
INFO: adding runtime plugin filemanager
INFO: adding runtime plugin sscanf
INFO: starting ./samp03svr in /home/codeah/Desktop/samp-sandbox
INFO: running with mode server
```

**Comment by Codeahh** [comment]
I have somehow managed to resolve the issue by deleting the project locally, and cloning samp-sandbox again.

**Comment by Southclaws** [comment]
Should be resolved in 1.8.10+
